com.unity.cinemachine/Runtime/Behaviours/CinemachineVirtualCamera.cs

```diff
@@ -622,5 +622,13 @@ internal override bool RequiresUserInput()
                 return true;
             return m_ComponentPipeline != null && m_ComponentPipeline.Any(c => c != null && c.RequiresUserInput);
         }
+
+        public void OnBeforeSerialize()
+        {
+            if (!m_Lens.IsPhysicalCamera) 
+                m_Lens.SensorSize = Vector2.zero;
```

Comment:
Don't use zero!  Use 1, 1 so that calculations won't blow up if it gets deserialized accidentally